lms/static/sass/elements/_course-card.scss

```diff
@@ -1,8 +1,46 @@
-.header-img{
-  @include float(left);
-  margin-right:20px;
-}
 .course-card{
-  height: 250px;
-  margin-top:20px;
+  width: 240px;
+  @include susy-media($bp-screen-sm) { width: 240px; }
+  @include susy-media($bp-screen-md) { width: 700px; }
+  @include susy-media($bp-screen-lg) { width: 920px; }
+  margin-left: $baseline*2 + px;
+  margin-bottom: $baseline + px;
+  .course-image-link{
+    .header-image{
+      width: 191px;
+      height: 100px;
```

Comment:
The image dimensions in production are not standardized so setting the width and height with distort some images because their aspect ratio will change. I think we should just set the width and the height will automatically be set accordingly.